stable/mongodb-replicaset/templates/mongodb-statefulset.yaml

```diff
@@ -241,7 +241,7 @@ spec:
             - -c
             - >-
               /bin/mongodb_exporter
-              --mongodb.uri {{ template "mongodb-replicaset.connection-string" . }}
+              --mongodb.uri "{{ template "mongodb-replicaset.connection-string" . }}"
```

Comment:
Instead of quotes within quotes, it would be better to pipe template to quote function.

`{{ template "mongodb-replicaset.connection-string" . | quote }}`

Reply:
Thanks for the suggestion.  I tried making the changes you suggested, but I get a compilation error:
```
Error: render error in "mongodb-replicaset/templates/mongodb-statefulset.yaml": template: mongodb-replicaset/templates/_helpers.tpl:67:16: executing "mongodb-replicaset.connection-string" at <.Values.auth.enabled>: can't evaluate field Values in type string
```

I grep'd the charts repo for other instances where one pipes template variables, but I couldn't find any other examples.  I'm not sure if it's possible to do so (also, see double quotes around METRICS_* env variables, which we'd also want to change if we could:
```
name: "{{ template "mongodb-replicaset.metricsSecret" . }}"
```